tests/services/deduplication/test_dedup_semantic.py

```
@@ -434,3 +434,37 @@ def test_stats_values_reasonable(self, analyzer, sample_embeddings):
         assert 0.0 <= stats["max_similarity"] <= 1.0
         assert stats["min_similarity"] >= 0.0
         assert stats["above_threshold_count"] >= 0
+
+    @pytest.mark.ci
+    def test_single_document_corpus(self, analyzer):
+        """get_statistics on a 1×1 matrix returns zeros without raising (issue #100)."""
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)